Mathlib/Data/String/Basic.lean

```diff
@@ -0,0 +1,333 @@
+/-
+Copyright (c) 2018 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro
+
+! This file was ported from Lean 3 source module data.string.basic
+! leanprover-community/mathlib commit 8a275d92e9f9f3069871cbdf0ddd54b88c17e144
```

Comment:
This commit doesn't exist: https://github.com/leanprover-community/mathlib/commit/8a275d92e9f9f3069871cbdf0ddd54b88c17e144